.github/workflows/update-downstream.yml

```diff
@@ -37,6 +37,18 @@ jobs:
       TARGET_REPO: ${{ github.repository_owner }}/${{ matrix.repo }}
       GH_TOKEN: ${{ secrets.detsys_pr_bot_token }} # for `gh pr create`
     steps:
+      - name: Check out detsys-ts
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ github.repository }}
+          token: ${{ env.GH_TOKEN }}
+
+      - name: Get last commit message
+        id: commit-msg
+        run: |
+          MSG=$(git show -s --format="%s" ${{ github.sha }}))
+          echo "msg=${MSG}" >> $GITHUB_OUTPUT
+
       - name: Check out ${{ env.TARGET_REPO }}
         uses: actions/checkout@v4
         with:
```

Comment:
This seems like it'll behave weirdly. checkout documents a few methods of how to check out multiple repos, and they all require a path:

* https://github.com/actions/checkout?tab=readme-ov-file#checkout-multiple-repos-side-by-side
* https://github.com/actions/checkout?tab=readme-ov-file#checkout-multiple-repos-nested
* https://github.com/actions/checkout?tab=readme-ov-file#checkout-multiple-repos-private

Reply:
Okay, I've split this into two separate dirs. I *think* this should work but curious what you think.